<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SLPreferenceDialog</class>
 <widget class="QDialog" name="SLPreferenceDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>341</width>
    <height>545</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Preferences</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>510</y>
     <width>261</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QLabel" name="cameraLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>120</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Camera:</string>
   </property>
  </widget>
  <widget class="QLabel" name="projectorLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>131</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Projector:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="cameraComboBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>140</y>
     <width>321</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="patternModeLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>121</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Pattern Mode:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="patternModeComboBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>321</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="projectorComboBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>80</y>
     <width>321</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="triggerModeLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>210</y>
     <width>121</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Trigger Mode:</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>380</y>
     <width>261</width>
     <height>24</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="triggerModeLayout">
    <item>
     <widget class="QRadioButton" name="triggerHardwareRadioButton">
      <property name="text">
       <string>Hardware</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="triggerSoftwareRadioButton">
      <property name="text">
       <string>Software</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="writeToDiskLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>270</y>
     <width>131</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Write to disk:</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>180</y>
     <width>201</width>
     <height>29</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="shutterLayout">
    <item>
     <widget class="QLabel" name="shutterLabel">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="text">
       <string>Shutter:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QDoubleSpinBox" name="shutterDoubleSpinBox">
      <property name="decimals">
       <number>3</number>
      </property>
      <property name="maximum">
       <double>999.990000000000009</double>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="delayMsLabel_2">
      <property name="text">
       <string>ms</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>230</y>
     <width>261</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="triggerLayout">
    <item>
     <layout class="QHBoxLayout" name="delayLayout">
      <item>
       <widget class="QLabel" name="delayLabel">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="text">
         <string>Delay:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="delaySpinBox">
        <property name="maximum">
         <number>9999</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="delayMsLabel">
        <property name="text">
         <string>ms</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>290</y>
     <width>286</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <item row="0" column="0">
     <widget class="QCheckBox" name="framesCheckBox">
      <property name="text">
       <string>Captured Frames</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QCheckBox" name="pointCloudsCheckBox">
      <property name="text">
       <string>Point Clouds</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>220</y>
     <width>251</width>
     <height>24</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="diamondPatternLayout">
    <item>
     <widget class="QCheckBox" name="diamondPatternCheckBox">
      <property name="text">
       <string>Diamond Pixel Pattern</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="aquisitionLabel">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>10</y>
     <width>121</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Aquisition:</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget_2">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>30</y>
     <width>271</width>
     <height>24</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="aquisitionLayout">
    <item>
     <widget class="QRadioButton" name="aquisitioncontinuousRadioButton">
      <property name="text">
       <string>Continuous</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="aquisitionSingleRadioButton">
      <property name="text">
       <string>Single Point Cloud</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_3">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>120</y>
     <width>261</width>
     <height>24</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="patternModeLayout">
    <item>
     <widget class="QCheckBox" name="patternHorizontalCheckBox">
      <property name="text">
       <string>Horizontal</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="patternVerticalCheckBox">
      <property name="text">
       <string>Vertical</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_4">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>350</y>
     <width>271</width>
     <height>24</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="aquisitionLayout_2">
    <item>
     <widget class="QRadioButton" name="flipcamera0">
      <property name="text">
       <string>LeftCam0</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="flipcamera1">
      <property name="text">
       <string>RightCam1</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="writeToDiskLabel_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>330</y>
     <width>31</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Flip: </string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget_5">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>410</y>
     <width>279</width>
     <height>29</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="aquisitionLayout_3">
    <item>
     <widget class="QLabel" name="shutterLabel_2">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="text">
       <string>Threshold Complementary:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QDoubleSpinBox" name="Threshold_Complentary">
      <property name="decimals">
       <number>3</number>
      </property>
      <property name="maximum">
       <double>1.000000000000000</double>
      </property>
      <property name="singleStep">
       <double>0.010000000000000</double>
      </property>
      <property name="value">
       <double>0.010000000000000</double>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="aquisitionLabel_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>390</y>
     <width>291</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Decoding:(Default: 0.01)</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget_6">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>470</y>
     <width>294</width>
     <height>29</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="aquisitionLayout_4">
    <item>
     <widget class="QLabel" name="shutterLabel_3">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="text">
       <string>Threshold Epipolar Line Width:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QDoubleSpinBox" name="Threshold_Epipolarline">
      <property name="decimals">
       <number>3</number>
      </property>
      <property name="maximum">
       <double>10.000000000000000</double>
      </property>
      <property name="singleStep">
       <double>0.500000000000000</double>
      </property>
      <property name="value">
       <double>1.000000000000000</double>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="aquisitionLabel_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>450</y>
     <width>311</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Triangulator: (Default: 1.000)</string>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>420</y>
     <width>112</width>
     <height>29</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="shiftLayout">
    <item>
     <widget class="QSpinBox" name="shiftSpinBox">
      <property name="minimum">
       <number>0</number>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="shiftLabel">
      <property name="text">
       <string>Shift:</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QCheckBox" name="trackingCheckBox">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>470</y>
     <width>131</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Tracking Data</string>
   </property>
  </widget>
  <zorder>layoutWidget</zorder>
  <zorder>layoutWidget</zorder>
  <zorder>layoutWidget</zorder>
  <zorder>layoutWidget</zorder>
  <zorder>layoutWidget</zorder>
  <zorder>buttonBox</zorder>
  <zorder>cameraLabel</zorder>
  <zorder>projectorLabel</zorder>
  <zorder>cameraComboBox</zorder>
  <zorder>patternModeLabel</zorder>
  <zorder>patternModeComboBox</zorder>
  <zorder>projectorComboBox</zorder>
  <zorder>triggerModeLabel</zorder>
  <zorder>writeToDiskLabel</zorder>
  <zorder>aquisitionLabel</zorder>
  <zorder>layoutWidget_2</zorder>
  <zorder>layoutWidget_3</zorder>
  <zorder>layoutWidget_4</zorder>
  <zorder>writeToDiskLabel_2</zorder>
  <zorder>layoutWidget_5</zorder>
  <zorder>aquisitionLabel_2</zorder>
  <zorder>layoutWidget_6</zorder>
  <zorder>aquisitionLabel_3</zorder>
  <zorder></zorder>
  <zorder>trackingCheckBox</zorder>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>SLPreferenceDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>SLPreferenceDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
